templates/editCounter/general_stats.html.twig

```diff
@@ -514,34 +514,20 @@
                 )
             }}
         {% endif %}
-        {{
-            chart.pie_chart('small_edits',
-                [{
-                    label: '< ' ~ 20|num_format ~ ' ' ~ msg('num-bytes', [20]) ~ '*',
-                    value: ec.countSmallEdits,
-                    percentage: ((ec.countSmallEdits / ec.countLast5000) * 100)
-                },
-                {
-                    label: '≥ ' ~ 20|num_format ~ ' ' ~ msg('num-bytes', [20]) ~ '*',
-                    value: ec.countLast5000 - ec.countSmallEdits,
-                    percentage: 100 - ((ec.countSmallEdits / ec.countLast5000) * 100)
-                }]
-            )
-        }}
-        {{
-            chart.pie_chart('large_edits',
-                [{
-                    label: '< ' ~ 1000|num_format ~ ' ' ~ msg('num-bytes', [1000]) ~ '*',
-                    value: ec.countLast5000 - ec.countLargeEdits,
-                    percentage: 100 - ((ec.countLargeEdits / ec.countLast5000) * 100)
-                },
-                {
-                    label: '≥ ' ~ 1000|num_format ~ ' ' ~ msg('num-bytes', [1000]) ~ '*',
-                    value: ec.countLargeEdits,
-                    percentage: ((ec.countLargeEdits / ec.countLast5000) * 100)
-                }]
-            )
-        }}
+        <div style="float:left">{{ msg('num-edits-size-interval') }}*
```

Comment:
This gets flipped in RTL. You can use the [Bootstrap](https://getbootstrap.com/docs/3.4/css/#helper-classes-floats) `pull-left` class and that will get flipped by the RTL stylesheet.

We should instead be using [CSSJanus](https://www.npmjs.com/package/cssjanus), but that might necessitate [T392531](https://phabricator.wikimedia.org/T392531). Anyway, not for this PR.